projects/onto-search/src/lib/onto-search-facet/range-facet/range-facet.component.html

```diff
@@ -0,0 +1,4 @@
+<div class="range-facet" appCypressData="onto-range-facet">
+<onto-range-histogram *ngIf="data.showHistogram" [data]="data" [containerWidth]="containerWidth"></onto-range-histogram>
+<onto-range-slider [data]="data" (sliderWidth)="onSliderWidth($event)" (selectionChange)="onSelectionEvent($event)"></onto-range-slider>
```

Comment:
`sliderWidth` as event name sounds really strange.

Reply:
My bad. It's actually the opossite. It's the event fired when slider's container width gets changed.

Reply:
renamed